synapse_net/inference/util.py

```diff
@@ -123,9 +123,12 @@ def get_prediction(
     # If we have channels then the standardization is done independently per channel.
     if with_channels:
         input_volume = input_volume.astype(np.float32, copy=False)
+        channels_to_standardize = None
```

Comment:
Now you are over-riding your input so that it's always None. If you remove this line then everything should be fine.